sar/iq_correction.cu

```diff
@@ -323,3 +325,51 @@ void RawDataCorrection(DevicePaddedImage& img, CorrectionParams par, SARResults&
         }
     }
 }
+
+__global__ void CalibrateClampToBigEndian(cufftComplex* data, int x_size_stride, int x_size, int y_size,
```

Comment:
I think this would belong in a new file, e.g envisat_format_kernels.cu.

I/Q correction specifically refers to the thee corrections that are implemented here